cmd/aspect/main.go

```diff
@@ -50,15 +50,15 @@ func main() {
 	root.HandleVersionFlags(streams, os.Args[1:], bzl)
 
 	// Re-enter another aspect if version running is not the configured version
-	reentered, err := bzl.HandleReenteringAspect(streams, os.Args[1:], root.CheckAspectLockVersionFlag(os.Args[1:]))
+	bazelVersion, reentered, err := bzl.HandleReenteringAspect(streams, os.Args[1:], root.CheckAspectLockVersionFlag(os.Args[1:]))
```

Comment:
might make sense to expose this on the bazel interface instead. Up to yall, I don't feel strongly how this is spelled. This reentrancy bit is very complex FWIW :)